gepetto/ida/cli.py

```diff
@@ -77,6 +87,8 @@ def handle_response(response):
                         "tool_calls": tool_calls,
                     }
                 )
+                STATUS_PANEL.log(_("→ Model requested tool: {tool_name} ({tool_args}...)").format(tool_name=tc.function.name, tool_args=(tc.function.arguments or '')[:120]))
```

Comment:
The `tc` variable doesn't exist here, I suggest moving this right after the for loop on line 92